netbout-web/src/test/java/com/netbout/rest/bout/TkAttachTest.java

```diff
@@ -80,13 +86,23 @@ public void sendsMessageToBout() throws Exception {
             new RqMultipart.Fake(
                 TkAttachTest.fake(number),
                 new RqWithHeaders(
-                    new RqLive(
-                        new ByteArrayInputStream("content1".getBytes())
-                    ),
+                    new Request() {
```

Comment:
@exper0 we can move creation of the request into static method and use it here and in the `ignoresWrongFile`